ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -230,6 +247,32 @@ const ProfileMainScreenFC = () => {
       />
       <VSpacer size={8} />
       <ContentWrapper>
+        {appFeedbackEnabled && surveyUrl && (
+          <ListItemAction
+            label={feedbackLabel}
+            icon="message"
+            variant="primary"
+            testID="feedbackButton"
+            onPress={() => {
+              void openAuthenticationSession(surveyUrl, "");
+            }}
+            accessibilityLabel={feedbackLabel}
+          />
+        )}
+        <ListItemAction
+          label={reviewLabel}
+          icon="starEmpty"
+          variant="primary"
+          testID="reviewButton"
+          onPress={Platform.select({
+            ios: () =>
+              openWebUrl(
+                "https://apps.apple.com/app/id1501681835?action=write-review"
+              ),
+            default: requestAppReview
```

Comment:
For Android, if we use the native approach, we won’t receive any feedback when the review is not shown. Am I right? It this the expected behavior? I've got a Motorola that showed me the native prompt only the first time, and then nothing...

| Motorola | Honor |
|--------|--------|
| <video src="https://github.com/user-attachments/assets/e7174ce5-d170-4b72-9e45-a8c40cbf3ce5" /> | <video src="https://github.com/user-attachments/assets/80d2eedc-426f-4031-817f-03191177f754" /> |